### DISCUSSION TITLE
What data types are supported by term filtering?

### DISCUSSION THREAD

**Discussion by Wenorm** [milvus-io/milvus#4546] [Q&A and General discussion] [answered]
What data types are supported by the latest version v0.11.0 for term filtering? I did not find support for the string type in the java version of the SDK.

**Comment by sahuang** [comment]
There is a Enum DataType which lists all available data types.

**Reply by Wenorm** [reply]
就是枚举类型里面定义了字符串类型，但是实际上好像没有做支持呀，传输到服务端的grpc协议中没有string类型。

**Reply by seanlei** [reply]
DataType.String is not supported,which is also not include in protocol file milvus.proto
![image](https://user-images.githubusercontent.com/3304321/103342935-bb84fe00-4ac5-11eb-9ec5-c7ee8ca0f70c.png)

**Reply by sahuang** [reply]
This is an issue in SDK 0.9.1, where string type was included in DataType. In upcoming 0.9.2, it will be excluded.

**Comment by yhmo** [comment, marked as answer]
The DataType.String is pre-definition for future version. Currently, milvus server doesn't support string type field.
